code.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "name": "knoten",
+    "organization": "U.S. Geological Survey",
+    "description": "GitHub code repository for the knoten package",
+    "version": "0.2.1",
+    "status": "Pending",
+
+    "permissions": {
+      "usageType": "openSource",
+      "licenses": [
+        {
+          "name": "Public Domain, CC0-1.0",
+          "URL": "https://github.com/USGS-Astrogeology/knoten/blob/master/LICENSE.md"
+        }
+      ]
+    },
+
+    "homepageURL": "https://github.com/USGS-Astrogeology/knoten/releases/tag/0.1.0",
+    "downloadURL": "https://github.com/USGS-Astrogeology/knoten/archive/refs/tags/0.1.0.zip",
+    "disclaimerURL": "https://github.com/USGS-Astrogeology/knoten/blob/master/LICENSE.md",
```

Comment:
This needs to be point to the new DISCLAIMER.md file and not the LICENSE.md file

Reply:
This needs to be handle in ISIS as well

Reply:
PR is up for ISIS